recipes/openimageio/all/conanfile.py

```diff
@@ -96,6 +101,8 @@ def requirements(self):
             self.requires("fmt/9.1.0", transitive_headers=True)
 
         # Optional libraries
+        if self.options.with_libjxl:
+            self.requires("libjxl/0.10.3")
         if self.options.with_libpng:
             self.requires("libpng/1.6.42")
```

Comment:
We can set this to a version range and close https://github.com/conan-io/conan-center-index/pull/23122 in the process

Reply:
Just added the version range as suggested.

Reply:
Ups, somehow I seem to have accidentially lost the recent change here. Will fix that quickly